src/libPMacc/include/algorithms/math/doubleMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<double>
     }
 };
 
+template<>
+struct Float2int_rn<double>
+{
+    typedef int result;
```

Comment:
It probably would, but it is not possible ATM: The functions in in PMacc (https://github.com/ComputationalRadiationPhysics/picongpu/blob/release-0.1.1/src/libPMacc/include/algorithms/math/defines/floatingPoint.hpp#L41) expect it that way: `typename Floor< T1>::result`
Syntax highlighting helps ;-) But I'd also like to change it to at least `Result` to mark it as a type.
@ax3l @psychocoderHPC : What do you think?